[IMP] sales: remove delivery method features (#17911)
odoo/documentation: content/applications/sales/sales/sales_quotations/orders_and_variants.rst

Reviewer: If it needs to be enabled first, then that instruction should be included with the other setting. It gets lost in a note.

Reviewer: This is something I learned early in my career, and it stands out to me when I review content...

The "/" is ambiguous. I end up questioning: Do we mean "and"? Do we mean "or"?

Do you think we can get away with saying "selected or added" or even just "selected"?

Reviewer: ```suggestion
variants and add them to a quotation or sales order in a single view.
```

entry*.

.. note::
   It should be noted that the setting is titled, *Variant Grid Entry* on the **Sales** app settings

>    page, and titled, *Order Grid Entry* on product forms. So, be sure to keep that in mind.

Reviewer: ```suggestion
   It should be noted that the setting is titled *Variant Grid Entry* on the **Sales** app settings
page and titled *Order Grid Entry* on product forms. Be sure to keep that in mind.
```

There are comma splices here. I'm also going to make a suggestion that is a bit more direct, and you can use it if you feel like it still works...I'd replace the whole note with:

`This setting is labeled as *Variant Grid Entry* within the **Sales** app settings, but appears as *Order Grid Entry* on individual product forms.`

Settings
========

> When working with product variants, Odoo uses the product configurator, by default. To add the

Reviewer: ```suggestion
When working with product variants, Odoo uses the product configurator by default. To add the
```

variant grid entry option, that feature **must** be enabled in the Odoo **Sales** application. The
variant grid entry option provides a pop-up window on the quotation/sales order to simplify the
variant selection process.

To enable that setting, go to the :menuselection:`Sales app --> Configuration --> Settings`, and

> scroll to the *Product Catalog* section. Then, check the box next to the :guilabel:`Variant Grid

Reviewer: Just a quibble: The user doesn't even have to scroll; it's right at the top of the Settings page. I'd argue we don't need to add the "scroll" part; just say `in the *Product Catalog* section, select the checkbox next to the...` and complete it from there.

> Entry` option, and click :guilabel:`Save`.

Reviewer: ```suggestion
To enable that setting, go to the :menuselection:`Sales app --> Configuration --> Settings`, then
scroll to the *Product Catalog* section. Then, select the checkbox next to the :guilabel:`Variant Grid
Entry` option and click :guilabel:`Save`.
```

"Check the box" does not translate very well, so I've corrected it here. There are also extra commas, so I removed them or changed wording to make the comma work.

.. image:: orders_and_variants/variant-grid-entry-setting.png

>    :alt: The Product Catalog section showing the Variants and Variant Grid Entry options enabled.

Reviewer: I'd personally argue that because you don't even have to scroll to get to this setting, you can eliminate it entirely.

Configurator` in a table layout, allowing the user to select larger numbers of unique product
variants and add them to a quotation or sales order in a single view.

Product configurator
====================

If the :guilabel:`Product Configurator` option is selected on the product form, the feature appears
as a *Configure your product* pop-up window when the product with **at least two** variants is added
to a quotation or sales order. The :guilabel:`Product Configurator` is the default option for new
products after the :guilabel:`Variant Grid Entry` feature is enabled on the *Settings* page.

.. image:: orders_and_variants/product-configurator-window.png
   :alt: The product configurator pop-up window that appears on a quotation or sales order.

The :guilabel:`Product Configurator` option helps salespeople choose exactly which product variant
to add to the quotation or sales order using a format similar to online shopping.

.. note::
   All products configured with variants display a *Configure your product* pop-up window by
   default. This is true even with the :guilabel:`Variant Grid Entry` option disabled.

Order grid entry
================

> If the :guilabel:`Order Grid Entry` option is selected on a product's form, the order grid entry

Reviewer 2: ```suggestion
If the :guilabel:`Order Grid Entry` option is selected on a product form, the order grid entry
```

*Order Grid Entry*) become available on every product form.

To configure a product form to use either a product configurator or variant grid entry, start by

> navigating to the :menuselection:`Sales app --> Products --> Products` to view all the products in

Reviewer: Just as a general note:

I know that Grammarly suggests adding "the" before every menuselection because we include "app" in it, but this is one case in which I feel it's wrong. Our other documentation doesn't generally use "the" before it, and it feels clunky to me to add it. However, I feel this is more a matter of personal style, and if it feels right to you, you can totally keep it. Just my two cents.

the database.

> Then, select the desired product to configure, or click :guilabel:`New`, to create a new product

Reviewer: ```suggestion
Then, select the desired product to configure, or click :guilabel:`New` to create a new product
```

> from scratch. Once on the product form, click into the :guilabel:`Attributes \& Variants` tab, where

Reviewer: ```suggestion
from scratch. Once on the product form, click into the *Attributes & Variants* tab, where
```

Tabs are italicized.

product variants can be viewed, modified, and added.

At the bottom of the :guilabel:`Attributes \& Variants` tab, there is a *Sales Variant Selection*

> section with two options: :guilabel:`Product Configurator` and :guilabel:`Order Grid Entry`.

Reviewer: ```suggestion
At the bottom of the *Attributes & Variants* tab, there is a :guilabel:`Sales Variant Selection`
section with two options: :guilabel:`Product Configurator` and :guilabel:`Order Grid Entry`.
```

Tabs get italicized, and because this is a section they need to work in, I'd argue it gets a guilabel.

.. note::
   It should be noted that these options **only** appear if at least two values of an attribute have

>    been added to the record.

Reviewer: If this is the case, I'd add it outside the note, before the mention of that section of the tab. Something like, `Add at least two attributes and values to the product form. After at least two are added, the Product Configurator and Order Grid Entry options appear in the Sales Variant Selection section.`

Product configurator
====================

The product configurator feature appears as a *Configure your product* pop-up window, as soon as a
product with (at least two) variants is added to a quotation or sales order, but **only** if the
:guilabel:`Product Configurator` option is selected on its product form.

.. image:: orders_and_variants/product-configurator-window.png
   :alt: The product configurator pop-up window that appears on a quotation or sales order.

.. note::
   This *Configure your product* pop-up window also appears if the :guilabel:`Order Grid Entry`
   setting is **not** activated, as it is the default option Odoo uses when dealing with product

>    variants on quotations and/or sales orders.

Reviewer: ```suggestion
   The *Configure your product* pop-up window also appears if the :guilabel:`Variant Grid Entry`
   setting is **not** activated, as it is the default option Odoo uses when dealing with product
   variants on quotations and sales orders.
```

I assume we're talking about the Sales setting, so I changed that language here to match. It could cause confusion about why we're talking about this setting on the product form when we seem to mean the Sales setting.

Author: No, we are talking about the setting on the product form, which is called by a different name. I will review this section to ensure that it is clear.

> The :guilabel:`Product Configurator` option lets salespeople choose exactly which product variant to

Reviewer: ```suggestion
The :guilabel:`Product Configurator` option helps salespeople choose exactly which product variant to
```

I feel this is clearer.

add to the quotation or sales order using a format similar to online shopping.

Order grid entry
================

The order grid entry feature appears as a *Choose Product Variants* pop-up window, as soon as a
product with (at least two) variants is added to a quotation or sales order, but **only** if the

> :guilabel:`Order Grid Entry` option is selected on its product form.

Reviewer: ```suggestion
If the :guilabel:`Order Grid Entry` option is selected on a product's form, the order grid entry feature appears as a *Choose Product Variants* pop-up window.
```

I think we can put the feature first and simplify this sentence. The option isn't available to users unless the product has at least two variants, as we mentioned elsewhere, so I've removed that from the sentence.

.. image:: orders_and_variants/choose-product-variants-popup.png
   :alt: The choose product variants pop-up window that appears on a quotation in Odoo.

The *Choose Product Variants* pop-up window features all the variant options for that particular
product. From this pop-up window, the salesperson can designate how many of each variant they'd like

> to add to the quotation/sales order at once.

Reviewer: ```suggestion
to add to the quotation or sales order at once.
```

Same note about the slash.

When all the desired quantities and variants have been selected, the salesperson clicks

> :guilabel:`Confirm`, and those orders are instantly added to the quotation/sales order in the

Reviewer: ```suggestion
:guilabel:`Confirm`, and those orders are instantly added to the quotation or sales order in the
```

Same note about the slash.
